engine/src/main/java/io/zeebe/engine/state/instance/ElementInstance.java

```diff
@@ -30,16 +29,18 @@
       new LongProperty("calledChildInstanceKey", -1L);
   private final ObjectProperty<IndexedRecord> recordProp =
       new ObjectProperty<>("elementRecord", new IndexedRecord());
+  private final IntegerProperty activeSequenceFlowProp =
+      new IntegerProperty("activeSequenceFlow", 0);
```

Comment:
We should use the plural form because we have multiple sequence flows.

Same for `incrementActiveSequenceFlow()` and `incrementActiveSequenceFlow()`.

```suggestion
  private final IntegerProperty activeSequenceFlowsProp =
      new IntegerProperty("activeSequenceFlows", 0);
```